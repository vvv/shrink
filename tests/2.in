a TEST 1
b TEST 2
c TEST 3
d TEST' 4
d TEST' 4
