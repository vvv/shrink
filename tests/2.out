a TEST 1
.. 2 more lines are pretty similar
d TEST' 4
d TEST' 4
